main.go

```diff
@@ -274,6 +299,39 @@ func (k *Kasse) Authenticate(username string, password []byte) (*User, error) {
 	return user, nil
 }
 
+func ResultToLCD(lcd *lcd2usb.Device, res *Result, err error) error {
+	var r, g, b uint8
+	var text string
+	if res != nil {
+		text = res.Display()
+		switch res.Code {
+		default:
+			r, g, b = 255, 255, 255
+		case PaymentMade:
+			r, g, b = 0, 255, 0
+		case LowBalance:
+			r, g, b = 255, 50, 0
+		case AccountEmpty:
+			r, g, b = 255, 0, 0
+		}
+	} else {
+		r, g, b = 255, 0, 0
+		text = err.Error()
+	}
+
+	log.Println(r, g, b)
+	lcd.Color(r, g, b)
+	for i, l := range strings.Split(text, "\n") {
+		lcd.CursorPosition(1, uint8(i+1))
+		fmt.Fprint(lcd, l)
+	}
+	// TODO: Make flag
```

Comment:
Because it means that if we notice after a month that the colors are indescernible, when the LED-bulbs are a deep violete, the sun shines in a 37° angle and someones blue shirt reflects in the display, we don't have to rebuild, -commit and -deploy the binary, but instead can just quickly remount / rw, change the service-file and change the config in the deploy script, which is simpler.